src/asana/controller.py

```diff
@@ -33,6 +35,14 @@ def update_task(pull_request: PullRequest, task_id: str):
     }
     asana_client.update_task(task_id, update_task_fields)
     asana_client.add_followers(task_id, fields["followers"])
+    maybe_complete_tasks_on_merge(pull_request)
+
+
+def maybe_complete_tasks_on_merge(pull_request: PullRequest):
+    if asana_logic.should_autocomplete_tasks_on_merge(pull_request):
+        task_ids_to_complete_on_merge = asana_helpers.get_linked_task_ids(pull_request)
+        for complete_on_merge_task_id in task_ids_to_complete_on_merge:
+            asana_client.update_task(complete_on_merge_task_id, {"completed": True})
```

Comment:
nit: could be nice to just have a `asana_client.complete_task` function rather than passing in this dict of `{"completed": True}` to `update_task`

Reply:
Also - what happens if the task is already complete? Would this noop or throw an exception?

Reply:
It's a noop